src/main/java/frc/team2412/robot/subsystem/DrivebaseSubsystem.java

```diff
@@ -42,16 +48,21 @@
                 new FeedforwardConstraint(3.0, FEEDFORWARD_CONSTANTS.getVelocityConstant(),
                         FEEDFORWARD_CONSTANTS.getAccelerationConstant(), false), // old value was 11.0
                 new MaxAccelerationConstraint(3.0), // old value was 12.5 * 12.0
-                new CentripetalAccelerationConstraint(3.0) // old value was 15 * 12.0
+                new CentripetalAccelerationConstraint(3.0), // old value was 15 * 12.0
+                // in inches
+                new FeedforwardConstraint(11.0, FEEDFORWARD_CONSTANTS.getVelocityConstant(),
+                        FEEDFORWARD_CONSTANTS.getAccelerationConstant(), false),
+                new MaxAccelerationConstraint(12.5 * 12.0),
+                new CentripetalAccelerationConstraint(15 * 12.0)
```

Comment:
aren't these added to trajectory constraints earlier? (feedforwardconstraint, maxaccelerationconstraint, etc.)

Reply:
No it's a different thing plus we are getting rid of constants class